addons/account/models/account_tax.py

```diff
@@ -1408,6 +1408,17 @@ def _fix_tax_included_price_company(self, price, prod_taxes, line_taxes, company
             line_taxes = line_taxes.filtered(lambda tax: tax.company_id == company_id)
         return self._fix_tax_included_price(price, prod_taxes, line_taxes)
 
+    @api.model
+    def _filter_taxes_by_company(self, tax_ids, company_id):
```

Comment:
```suggestion
    def _filter_taxes_by_company(self, company):
```
Why `api.model` if we always have to give taxes ?
It could be called on the taxes directly :thinking:

Reply:
Yes. You're right! :+1: 

It was on prevision of some special case, but in the end, it was not necessary.

I've updated the fix.

Thanks. :slightly_smiling_face: